CHANGES.rst

```diff
@@ -1,9 +1,31 @@
 Release Notes
 =============
 
+1.6.4
+-----
+
+This release un-breaks `lambdex <https://github.com/wickman/lambdex>`_.
+
+* Restore ``pex.pex_bootstrapper.is_compressed`` API. (#685)
+  `PR #685 <https://github.com/pantsbuild/pex/pull/685>`_
+
+* Add the version of pex used to build a pex to build_properties. (#687)
+  `PR #687 <https://github.com/pantsbuild/pex/pull/687>`_
+
+* Honor interpreter constraints even when PEX_PYTHON and PEX_PYTHON_PATH not set (#668)
+  `PR #668 <https://github.com/pantsbuild/pex/pull/668>`_
+
 1.6.3
 -----
 
+This release changes the behavior of the ``--interpreter-constraint`` option.
```

Comment:
I missed this warning when releasing 1.6.3. Backfilling here and already updated on the GitHub release: https://github.com/pantsbuild/pex/releases/tag/v1.6.3